src/extension.ts

```diff
@@ -14,89 +14,89 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 import * as vscode from 'vscode';
-import { fetchBuilds, Job, JobStatus } from './cloudbuild';
-import { GitRepo } from './git';
+import {fetchBuilds, Job, JobStatus} from './cloudbuild';
+import {GitRepo} from './git';
 import * as time from './time';
 
 
 export var refreshTicker: NodeJS.Timeout;
-export var statusBar: vscode.StatusBarItem | undefined;
+export var statusBar: vscode.StatusBarItem | undefined;
 
 /**
- * Activate is called whenever the current workspace contains a 
+ * Activate is called whenever the current workspace contains a
  * "cloudbuild.yaml" file.
- * 
- * After activation, the extension will periodically (1000ms) call the `gcloud` 
- * command to get the list of all the builds for the current branch, then it 
+ *
+ * After activation, the extension will periodically (1000ms) call the `gcloud`
+ * command to get the list of all the builds for the current branch, then it
  * will display the most recent build status in the status bar with :
  * - a $(circle-slash) if no build is found
  * - a $(check) if the most recent build is in success
  * - a $(x) if the most recent build is in failure
- * - a $(repo-sync) (spining) if the most recent build is pending or running 
- * 
- * The status bar element will also have a tooltip with the time elapsed between 
+ * - a $(repo-sync) (spinning) if the most recent build is pending or running
+ *
+ * The status bar element will also have a tooltip with the time elapsed between
  * now and the last build.
- * 
+ *
  * @param context the execution context of the calling VSCode instance.
  */
 export function activate(context: vscode.ExtensionContext) {
-	// Create the status bar item & position it
-	statusBar = vscode.window.createStatusBarItem(vscode.StatusBarAlignment.Left, 1000);
-	refreshTicker = setInterval(() => {
-		if (!vscode.workspace.rootPath) {
-			fetchBuilds("master")
-				.then((jobs) => refreshStatusBar(jobs));
-		} else {
-			let repo = new GitRepo(vscode.workspace.rootPath);
-			repo.getCurrentBranch()
-				.then(branch => fetchBuilds(branch))
-				.then(jobs => refreshStatusBar(jobs));
-		}
-	}, 1000);
+  // Create the status bar item & position it
```

Comment:
What is the convention you use here ? A standard (or at least well known) one ?

Reply:
Basically I'm following the Google [style guide](https://google.github.io/styleguide/jsguide.html) and having 2 spaces instead of 4 is a common practice in bigger projects (for readiness sake).

Reply:
Not bad, maybe we can use this as a standard format for the project.